src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -143,7 +143,13 @@ export default function TechnicalIntegration() {
   )
 
   useEffect(() => {
-    setTechUserProfiles(userProfiles)
+    if (userProfiles.length > 0) {
+      setTechUserProfiles(userProfiles)
+    } else {
+      // Set default value as "None" when user profiles don't have any roles
+      // Initially, value is "None"
+      setTechUserProfiles([technicalUserNone])
+    }
```

Comment:
better use ? : instead of if-else here:
``` 
setTechUserProfiles(userProfiles.length > 0 ? userProfiles : [technicalUserNone])
```

Reply:
Hi @oyo, 
I am using the `if` and `else` statements because of easy to read and understand the code.  Sometimes `?: ` will create an issue. 

But anyway I will make these changes. 🙂